src/postamble.js

```diff
@@ -94,6 +94,39 @@ if (memoryInitializer) {
 #endif
 #endif
 
+#if CYBERDWARF
```

Comment:
looks like a copy-paste of the mem init code? perhaps we should consider a refactoring to share? not sure if feasible.